From jeffm@suse.com  Fri Jun 18 17:07:06 2010
From: Jeff Mahoney <jeffm@suse.com>
Date: Wed, 09 Jun 2010 16:01:10 -0400
Subject: Staging: wlags49_h2: add missing <linux/string.h> for strlen
To: Greg KH <gregkh@suse.de>
Message-ID: <4C0FF306.9030605@suse.com>



On ia64, the build fails with incompatible implicit definition of strlen.
This patch adds the <linux/string.h> include to get the real prototype.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlags49_h2/wl_enc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/wlags49_h2/wl_enc.c
+++ b/drivers/staging/wlags49_h2/wl_enc.c
@@ -62,6 +62,7 @@
 /*******************************************************************************
  *  include files
  ******************************************************************************/
+#include <linux/string.h>
 #include <wl_version.h>
 
 #include <debug.h>
